ecmascript/codegen/macros/src/lib.rs

```diff
@@ -57,26 +51,21 @@ fn (&mut self, node: Node) -> Result;
         };
 
         Quote::new_call_site()
-            .quote_with(smart_quote!(
+            .quote_with(pmutil::smart_quote!(
```

Comment:
Can you change this to `smart_quote!` with `use pmutil::smart_quote;`?

Reply:
> I prefer `util/mod.rs` over `util.rs` if util has submodules. Can you change them?

This is a main feature of 2018 edition. It's designed to resolve `mod.rs` jumping out in every search result and it's hard to find out which module the `mod.rs` is actually pointing to. Actually it's the main reason of this PR.

If you prefer mod.rs, I think it's better to remove 2018 edition declaration and/or put this into `rustfmt.toml` as a hint for contributors

Reply:
oops, I replied in the wrong comment. But the content declares itself, I think.

Reply:
> This is a main feature of 2018 edition. It's designed to resolve mod.rs jumping out in every search result and it's hard to find out which module the mod.rs is actually pointing to. Actually it's the main reason of this PR

Oh.. I didn't know that. I agree that mod.rs showing everywhere is bit annoying.
I'll just change my editor's config to align files based on named, not type.

Reply:
Removing `mod.rs` leaves many submodules containing only a test file. That's kind of weird indeed. In the future we could split the tests into integration tests (move them into the related module files) and unit tests(move to `/tests` dir). That will make the project cleaner.